src/app/forms/components/brakes/brakes.component.html

```diff
@@ -36,12 +36,13 @@
           </td>
 
           <td class="govuk-table__cell">
-            <app-text-input
```

Comment:
There seems to be a typo on Lever length column header, and the validators are missing from the Lever length and Brake actuator fields

Reply:
Typo fixed and validators added. Thanks!